mmv1/products/documentaiwarehouse/Location.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Location'
+base_url: 'projects/{{project}}/locations/us:initialize'
+self_link: '{{name}}'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/document-warehouse/docs/overview'
+  api: 'https://cloud.google.com/document-warehouse/docs/reference/rest/v1/projects.locations'
+description: |
+  A location is used to initialize a project.
+skip_delete: true
+skip_sweeper: true
+# import_format:
+#   [
+#     'projects/{{project}}/locations/{{location}}',
+#     '{{location}}'
+#   ]
+# timeouts: !ruby/object:Api::Timeouts
+#   insert_minutes: 20
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "document_ai_warehouse_location"
+    primary_resource_id: "example"
+# custom_code: !ruby/object:Provider::Terraform::CustomCode
+#   custom_create: 'templates/terraform/custom_create/document_ai_warehouse_location.go'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    description: |
+      The location of the resource.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
+    description: |
+      Name of the instance. 
+  - !ruby/object:Api::Type::Enum
```

Comment:
Can you please add `required: true` to the field databaseType, as it is required based on the API doc? Thanks.